packages/mobile/src/identity/feelessVerification.ts

```diff
@@ -1029,6 +1036,13 @@ export function* feelessCompleteAttestation(
 
   Logger.debug(TAG + '@feelessCompleteAttestation', `Completing code for issuer: ${code.issuer}`)
 
+  // Needed to ensure that codes inputted in quick succession are not
+  // assigned the same nonce by Komenci
+  while (yield select(feelessProcessingInputCodeSelector)) {
+    yield delay(100)
```

Comment:
It may not be as simple as `yield take` given we want them to be taken one-by-one and there may be 2 codes "waiting". I've left a `TODO` to look into this later.